### PR TITLE
Added TLDR about updating

### PR DESCRIPTION
**Pull request by Kezxo**
Just a way to update for people who have the password set up already.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,13 @@ nano -w ~/.plexupdate
 sudo ./plexupdate.sh -a
 ```
 
+##Updating the script and Plex
```
Comment (mrworf) [inline README.md]: I would add "if you don't want to use the built-in updater"

Comment (Kezxo) [inline README.md]: Crap, I missed that this was an option.. Maybe this is TLDR is pointless?

**Comment by Kezxo** [comment]
I added the thing you suggested, at first i thought the built in updater was preferable so this TLDR was not necessary but it required you to use git the first time so it made sense after all. Also maybe it is experimental? Sorry for taking so long.

Is the autoupdater still experimental? Should we use it or wget?

**Comment by mrworf** [comment]
I should probably remove the experimental tag on it ,seems to work stable enough (no reported bugs).

**Comment by mrworf** [comment]
To be honest, the new section just says to do whatever the install did, so I don't think it adds that much.

**Comment by Kezxo** [comment]
Maybe not...
The issue I had is that I ran that the tldr again and had to reenter my plex details again before I figured out the better way to do it and I entered them for no reasons because I was not a plex pass back then.

After that read more then the TLDR and noticed that the script had a auto update function, started using that instead.

Maybe the TLDR should use git instead? Doesn't basically all linux flavors come with git? Then they can use the auto update function later if they wish.

The readme also never mentions ``-p``.

I made a shorter version of the TLDR that all Plex users can use to install and update. Maybe it needs another version for Plex pass users who want the more frequent updates but if you use a TLDR then you probably don't care.

Let me know what you think. :)

File: README.md [hunk 2]
```diff
@@ -75,14 +75,16 @@ Overall it tries to give you hints regarding why it isn't doing what you expecte
 - "kaka" is swedish for "cookie"
 
 # TL;DR
-
 Open a terminal or SSH on the server running Plex Media Center
+## First install
+```
+git clone https://github.com/mrworf/plexupdate.git
+chmod +x plexupdate/plexupdate.sh
```
Comment (mrworf) [inline README.md]: You actually get the chmod +x for free when you use git, so this line can be skipped.

**Comment by mrworf** [comment]
Looking good except for the chmod thing
